obelix/idefix/services/coinmarketcap/domiance.go

```diff
@@ -0,0 +1,126 @@
+package coinmarketcap
```

Comment:
The filename contains a typo :/

Reply:
abe53828c9a08ec39de710174a918af0909572c0